cluster-up/check.sh

```diff
@@ -18,34 +18,46 @@
 #
 
 set -e
+fail=''
```

Comment:
might be nicer to set 0 instead of empty string ?
and of course change the condition accordingly if so